tests/Database/DatabaseEloquentModelTest.php

```diff
@@ -818,6 +819,17 @@ public function testMorphOneCreatesProperRelation()
         $this->assertEquals('EloquentModelStub', $relation->getMorphClass());
     }
 
+    public function testCorrectMorphClassIsReturned()
+    {
+        Relation::morphMap([
+            'alias' => 'AnotherModel',
+        ]);
+        $model = new EloquentModelStub;
+        $this->assertEquals('EloquentModelStub', $model->getMorphClass());
+
+        Relation::morphMap([], false);
```

Comment:
you need to wrap the assertion in a try finally, because currently if the assertion fails, this line is never executed